agot-bg-game-server/src/common/ingame-game-state/westeros-game-state/clash-of-kings-game-state/ClashOfKingsGameState.ts

```diff
@@ -36,6 +41,10 @@ export default class ClashOfKingsGameState extends GameState<WesterosGameState,
 
     onServerMessage(message: ServerMessage) {
         this.childGameState.onServerMessage(message);
```

Comment:
This line should be put in the `else` of the condition after, like it is done in other `GameState` classes.